### PR TITLE
Add webhooks: HlsUploaded, HlsUploadCrashed

### PR DESCRIPTION
**Pull request by Karolk99**
## Acknowledging the stipulations set forth:
 - [ ] I hereby confirm that a Pull Request involving updates to the Software Development Kit (SDK) has been smoothly merged, currently awaits processing, or is otherwise deemed unnecessary in this context.
 - [ ] I also affirm that another Pull Request, specifically addressing updates to the documentation body (commonly referred to as 'docs'), has either been successfully incorporated, is in the process of review, or is considered superfluous under the prevailing circumstances.

### CODE REVIEW

File: lib/protos/jellyfish/server_notifications.pb.ex [hunk 1]
```diff
@@ -121,6 +121,22 @@ defmodule Jellyfish.ServerMessage.HlsPlayable do
   field :component_id, 2, type: :string, json_name: "componentId"
 end
 
+defmodule Jellyfish.ServerMessage.HlsUploaded do
```
Comment (roznawsk) [inline lib/protos/jellyfish/server_notifications.pb.ex:124]: Imo `HlsUploaded` may not be so descriptive. Perhaps we should name this `HlsUploadSuccess` to indicate directly that the upload was a success

File: test/jellyfish_web/integration/server_notification_test.exs [hunk 2]
```diff
@@ -35,6 +41,15 @@ defmodule JellyfishWeb.Integration.ServerNotificationTest do
 
   @source_uri "rtsp://placeholder-19inrifjbsjb.it:12345/afwefae"
 
+  @files ["manifest.m3u8", "header.mp4", "segment_1.m3u8", "segment_2.m3u8"]
+  @body <<1, 2, 3, 4>>
```
Comment (roznawsk) [inline test/jellyfish_web/integration/server_notification_test.exs:45]: Remove this attribute, also in `component_controller_test`

**Review by roznawsk — CHANGES_REQUESTED** [review]

File: test/jellyfish_web/integration/server_notification_test.exs (hunk 2 above)
Comment (Karolk99) [reply test/jellyfish_web/integration/server_notification_test.exs:45]: I cannot do that. Component controller tests if hls manager sends correct data to s3

File: lib/protos/jellyfish/server_notifications.pb.ex (hunk 1 above)
Comment (Karolk99) [reply lib/protos/jellyfish/server_notifications.pb.ex:124]: I think that `HlsUploaded` is a descriptive EVENT as `RoomCreated`, adding `Success` is redundant and incompatible with the rest of messages.

What do you think @Rados13

**Review by roznawsk — APPROVED** [review]

File: lib/protos/jellyfish/server_notifications.pb.ex (hunk 1 above)
Comment (Rados13) [reply lib/protos/jellyfish/server_notifications.pb.ex:124]: IMO, it is descriptive enough.

**Review by Rados13 — APPROVED** [review]

**Pull request merged**